This would cause unsolvable circular dependencies since the
feature modules would have to be linked after the Cil module,
but before the Feature_config module, which is not possible
when both of them are packed in one .cma / .cmxa.

Index: src/cil.mllib
--- src/cil.mllib.orig
+++ src/cil.mllib
@@ -29,7 +29,6 @@ Epicenter
 Errormsg
 Escape
 Expcompare
-Feature_config
 Formatcil
 Formatlex
 Formatparse
